### DISCUSSION TITLE
More transparency on currently closed-door discussions via discord

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#4916] [Suggestions]
Currently, there are limited access channels in discord available to a selected playerbase, in the means of representing the community towards the server staff.

Unfortunately, those discussions are taken in these channels and mostly finished in there. Further discussion e.g. via github discussions are not initiated.

Therefore, I would propose to change the channel visibility to public. Write access can and should stay as is to preserve the purpose of the channel. The benefits would be that the community interested into the future of ChromieCraft could read up on current discussions, forward ideas and opinions to the respective representatives with write access. Further, people could read up on arguments that let to changes or stopped ideas from being implemented.

**Comment by heyitsbench** [comment]
An example of a decision happening without the knowledge of the general player base would be the recent change to handedness for transmog weapons. Pretty much the moment it actually became public (which was when it was implemented on CC as far as I'm aware), it was met with backlash. If decisions like these are actually seen (not necessarily polled, as that's already been addressed) then feedback can actually be given, even if it doesn't end up influencing the decision.
Whether or not this particular change (transmog handedness) is going to be (or already has been) reverted is also not easily accessible information, as with #4926 the fact that it's a toggleable config was pointed out, but still wasn't said whether or not it's actually going to change. Without looking at the various module repos, something like that just slips through the cracks, and it should *not* be that way, at least IMO.
